client/src/modules/stock/entry/modals/lots.modal.js

```diff
@@ -14,68 +14,68 @@ function StockDefineLotsModalController(Instance, Notify, uiGridConstants, Data,
   vm.entryType = Data.entry_type;
   vm.gridApi = {};
   vm.isCostEditable = (vm.entryType !== 'purchase' && vm.entryType !== 'transfer_reception');
-  
+
   vm.gridOptions = {
-    appScopeProvider: vm,
-    enableSorting: false,
-    enableColumnMenus: false,
-    showColumnFooter: true,
-    fastWatch: true,
-    flatEntityAccess: true,
-    columnDefs: [
+    appScopeProvider : vm,
+    enableSorting : false,
+    enableColumnMenus : false,
+    showColumnFooter : true,
+    fastWatch : true,
+    flatEntityAccess : true,
+    columnDefs : [
       {
-        field: 'status',
-        width: 25,
-        displayName: '',
-        cellTemplate: 'modules/stock/entry/modals/templates/lot.status.tmpl.html'
+        field : 'status',
+        width : 25,
+        displayName : '',
+        cellTemplate : 'modules/stock/entry/modals/templates/lot.status.tmpl.html',
       },
 
       {
-        field: 'lot',
-        displayName: 'TABLE.COLUMNS.LOT',
-        headerCellFilter: 'translate',
-        aggregationType: uiGridConstants.aggregationTypes.count,
-        aggregationHideLabel: true,
-        cellTemplate: 'modules/stock/entry/modals/templates/lot.input.tmpl.html'
+        field : 'lot',
+        displayName : 'TABLE.COLUMNS.LOT',
+        headerCellFilter : 'translate',
+        aggregationType : uiGridConstants.aggregationTypes.count,
+        aggregationHideLabel : true,
+        cellTemplate : 'modules/stock/entry/modals/templates/lot.input.tmpl.html',
       },
 
       {
-        field: 'quantity',
-        type: 'number',
-        width: 150,
-        displayName: 'TABLE.COLUMNS.QUANTITY',
-        headerCellFilter: 'translate',
-        aggregationType: uiGridConstants.aggregationTypes.sum,
-        aggregationHideLabel: true,
-        footerCellClass: 'text-right',
-        cellTemplate: 'modules/stock/entry/modals/templates/lot.quantity.tmpl.html'
+        field : 'quantity',
+        type : 'number',
+        width : 150,
+        displayName : 'TABLE.COLUMNS.QUANTITY',
+        headerCellFilter : 'translate',
+        aggregationType : uiGridConstants.aggregationTypes.sum,
+        aggregationHideLabel : true,
+        footerCellClass : 'text-right',
+        cellTemplate : 'modules/stock/entry/modals/templates/lot.quantity.tmpl.html',
       },
 
       {
-        field: 'expiration_date',
-        type: 'date',
```

Comment:
Note: we should add the bhConstants date format parameter here to ensure it renders the same across French/English clients.